src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler/Configuration.php

```diff
@@ -96,7 +96,7 @@ private function __construct(array $thresholds = [], $regex = '', $verboseOutput
         }
         foreach ($groups as $group) {
             if (!isset($this->thresholds[$group])) {
-                $this->thresholds[$group] = 999999;
+                $this->thresholds[$group] = 0;
```

Comment:
well, I'd rather add this after this foreach:
```php
$this->thresholds = $this->thresholds ?: ['total' => 0];
```

Reply:
If we have entered the constructor with an empty $thresholds array, at that point after the foreach, if I am not mistaken, `$this->thresholds` will be something like

```
[
  'total' => 999999,
  'indirect' => 999999, 
  'direct' => 999999, 
  'self' => 999999,
]
```

no?

Reply:
right! we should add this at the very beginning of the constructor then:
`$thresholds = $thresholds ?: ['total' => 0];`

Reply:
Done, thanks